README.md

```diff
@@ -119,62 +121,62 @@ Supported PHP extensions
-         mcrypt |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-          mhash |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |         |
-          mysql |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |         |
-         mysqli |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-        mysqlnd |         |         |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-        openssl |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-          pcntl |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-           pcre |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-            pdo |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-      pdo_mysql |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-      pdo_pgsql |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-     pdo_sqlite |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-          pgsql |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-           phar |         |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-          posix |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-     reflection |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-        session |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-      simplexml |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-           soap |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-        sockets |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-            spl |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-         sqlite |    ✓    |    ✓    |         |         |         |         |
-        sqlite3 |         |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-       standard |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-      tokenizer |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-           wddx |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-         xdebug |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-            xml |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-      xmlreader |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-      xmlwriter |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-            xsl |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-            zip |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
-           zlib |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓    |    ✓
+Extension | PHP 5.1 | PHP 5.2 | PHP 5.3 | PHP 5.4 | PHP 5.5 | PHP 5.6 | PHP 7.0 | PHP 7.1
+----------:|:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|:-------:
```

Comment:
Yes I manually adjusted it so it would show as a table in Markdown. Let me modify _extensions.php_ so it generates a Markdown table properly... will get back to you.